docs/products/clickhouse/reference/supported-table-functions.rst

```diff
@@ -10,13 +10,19 @@ Table functions supported in Aiven for ClickHouse®
       SELECT *
       FROM deltaLake('s3://bucket/path/to/lake')
 
+.. note::
+
+   You may find some specific table functions disabled for security reasons.
+
 Aiven for ClickHouse® supports the following table functions:
 
 * ``cluster``
 * ``clusterAllReplicas``
 * ``cosn``
 * ``deltaLake``
+* ``dictionary``
```

Comment:
We don't allow dictionaries so this is a bit moot

Reply:
you are right. mentioning it there will not help anyone. Deleted